src/main/java/games/strategy/engine/framework/startup/ui/editors/EmailSenderEditor.java

```diff
@@ -167,8 +167,9 @@ private void testEmail() {
         message = "Email sent, it should arrive shortly, otherwise check your spam folder";
         messageType = JOptionPane.INFORMATION_MESSAGE;
       } catch (final IOException ioe) {
-        message = "Unable to send email; see TripleA Console for details.";
-        ClientLogger.logError(ioe);
+        message = "Unable to send email, check SMPT server credentials: " +
+            truncate(ioe.getMessage(), 200);
```

Comment:
SMPT -> SMTP

Reply:
d'oh, yup, it is transport protocol, not protocol transport. Fixed in: f7d76f9d1